assignments/objects.js

```diff
@@ -49,16 +96,37 @@ const example = {
 // 3. Nest a grandchild object in the child object with properties for name and age.  The name will be Sam and the age will be 30
 // 4. Give each of the objects the ability to speak their names using the this keyword.
 
-const parent = {}
+const parent = {
+  name: 'Susan',
+  age: 70,
+  speak: function() {
+    console.log(`My name is: ${this.name}!`)
+  },
+    child: {
+    name: 'George',
+    age: 50,
+    speak: function() {
+      console.log(`My name is: ${this.name}!`)
+    },
+      grandchild: {
+      name: 'Sam',
+      age: 30,
+      speak: function() {
+        console.log(`My name is: ${this.name}!`)
+      },
+    }
+  }
+}
 
 // Log the parent object's name
-
+console.log (parent.name)
 // Log the child's age
-
+console.log (parent.child.name);
 // Log the name and age of the grandchild
-
```

Comment:
Log the name and age by specifying the key name and age not returning the grandchild object

Reply:
didn't work on this one